discovery/syncer.go

```diff
@@ -948,7 +955,7 @@ func (g *GossipSyncer) processChanRangeReply(msg *lnwire.ReplyChannelRange) erro
 // party when we're kicking off the channel graph synchronization upon
 // connection. The historicalQuery boolean can be used to generate a query from
 // the genesis block of the chain.
-func (g *GossipSyncer) genChanRangeQuery(
+func (g *GossipSyncer) genChanRangeQuery(_ context.Context,
```

Comment:
it seems we have a few occurrences of unhandled contexts, is it a TODO for future PRs?

Reply:
yeah if you look at these methods where im now pasing in `_ context.Context` - somewhere down in the method there will be a graph DB call that it will later be passed to